SCAutolib/models/gui.py

```diff
@@ -119,10 +175,15 @@ def move(self, x: float, y: float):
         self.device.syn()
 
     def click(self, button: str = 'left'):
-        """Clicks any button of the mouse.
-
-        :param button: mouse button to click, defaults to 'left'
-            Possible values 'left', 'right' or 'middle'.
+        """
+        Simulates a click of a specified mouse button.
+
+        :param button: The mouse button to click. Possible values are ``left``,
+                       ``right``, or ``middle``. Defaults to ``left``.
+        :type button: str
+        :return: None
+        :rtype: None
+        :raises KeyError: If button is not ``left``, ``right``, or ``middle``.
```

Comment:
Is `device.emit()` what raises KeyError?    Should this be mentioned here or maybe just a comment above the first `device.emit()` line?

Reply:
I do not think it needs additional comment. @spoore1 ?